src/tools/fuzzing.h

```diff
@@ -783,10 +898,81 @@ class TranslateToFuzzReader {
     ReFinalize().walkFunctionInModule(func, &wasm);
   }
 
+  void modifyInitialFunctions() {
+    if (wasm.functions.empty()) {
+      return;
+    }
+    // Pick a chance to fuzz the contents of a function.
+    const int RESOLUTION = 10;
+    auto chance = upTo(RESOLUTION + 1);
```

Comment:
Is `chance` the same across all functions? (Not that it's a problem, but just curious if you intended it to be within the loop or not)

Reply:
Yes, basically this makes it so sometimes we fuzz most functions, and sometimes few (and not always around the same % in a large enough module due to statistics). Not sure it matters much though.